docs/rawdata/raw_data_upload.md

```diff
@@ -33,23 +33,23 @@ Uploading your files to us was never this easy! SFTP is a broadly used file tran
 wide-spread use ensures that there exists many client software products that
 support uploading files to us.
 In this section we will go through the process of connecting to our server
-using [FileZilla](https://filezilla-project.org) as an example.
+using the [FileZilla](https://filezilla-project.org/download.php?type=client) client as an example.
```

Comment:
```suggestion
using the [FileZilla](https://filezilla-project.org) client as an example.
```

Reply:
The download button however does not specify which software they need (ftp client). The php link ensures that they get the correct ftp client for their OS